.eslintrc

```diff
@@ -6,8 +6,9 @@
       "ignoreUrls": true,
       "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
     }],
-    // NOTE(bajtos) we should eventuall remove this override
+    // NOTE we should eventually remove this override
     // and fix all of those 100+ violations
-    "one-var": "off"
+    "one-var": "off",
```

Comment:
I believe we don't need this rule override anymore, please confirm and send another PR to remove it if linting works without it.